File: 168 Excel Sheet Column Title/solution.cpp
class Solution {
public:
    string convertToTitle(int n) {
        string result = "";
        int tmp = n;
        int count = 0;
        while (tmp>=1){
            count++;
            tmp = (tmp-1)/26;
        }
        for (int i=0;i<count;i++){
            result+=('A'+((n-1)%26));
            n = (n-1)/26;
        }
        string strresult = "";
        int len = result.size();
        for (int i=len-1;i>=0;i--){
            strresult+=(result[i]);
        }
        return strresult;
    }
};

File: 168 Excel Sheet Column Title/solution.py
class Solution:
    # @param {integer} n
    # @return {string}
    def convertToTitle(self, n):
        result = ""
        tmp = n
        count = 1
        while (int((tmp-1)/26)>=1):
            count+=1
            tmp=int((tmp-1)/26)
        for i in range(count):
            result+=chr(((n-1)%26)+ord('A'))
            n = int((n-1)/26)
        return result[::-1]